runtime/redirector/redirector.c

```diff
@@ -491,69 +548,24 @@ chooseJVM(JavaVMInitArgs *args, char *retBuffer, size_t bufferLength)
 	int ignoreUnrecognizedEnabled = 0;
 
 	int gcPolicy = GC_POLICY_GENCON;
-	const char *gcPolicyOption = "-Xgcpolicy:";
-	size_t gcPolicyOptionLength = strlen(gcPolicyOption);
-	char *gcPolicyString = NULL;
 
 	char *xmxstr = NULL;
 	U_64 requestedHeapSize = 0;
 
-	/* the command line is handled below but look into the ENVVAR_OPENJ9_JAVA_OPTIONS here, since it is a special case */
+	/* 
+	 * The command line is handled below but look into the *JAVA_OPTIONS here, since it is a special case.
```

Comment:
Is the `*` in `*JAVA_OPTIONS` significant?

Reply:
Yes. I've rewritten the comment to be explicit.